modules/v2x/v2x_proxy/app/v2x_proxy.cc

```diff
@@ -57,7 +57,8 @@ V2xProxy::V2xProxy()
   }
 
   hdmap_.reset(new apollo::hdmap::HDMap());
-  std::string map_name = FLAGS_map_dir + "/" + FLAGS_base_map_filename;
+  std::string map_name = absl::StrCat(absl::GetFlag(FLAGS_map_dir), "/",
+                                      absl::GetFlag(FLAGS_base_map_filename));
```

Comment:
Seems like we have a lot of operations on file path.
Maybe we can add a "modules/common/util/file.h" later to provide things similar to python's
```
os.path.join()
os.path.basename()
...
```